### ISSUE TITLE
Add support for Dodo Payments

### ISSUE THREAD

**Issue by aagarwal1012** [developeratul/lemonsqueezy-nextjs#1]
Please add support for https://dodopayments.com

**Comment by developeratul** [comment]
We can collaborate on an article in that case. LMK if you're interested

**Comment by aagarwal1012** [comment]
Yes, we can do the same -- let me know what resources do you need from my
side.

**Comment by developeratul** [comment]
I charge $50 per blog. It will be posted on my Dev.to account. Same as this one. LMK if you want to proceed.
